crypto/digestible/signature/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2018-2020 MobileCoin Inc.
+
+#![no_std]
+
+use mc_crypto_digestible::{Digestible, MerlinTranscript};
+use signature::{Signature, Signer, Verifier};
+
+pub use signature::Error;
+
+/// Construct a deterministic semantic signature over a given object.
+pub trait DigestibleSigner<S: Signature, T: Digestible> {
+    fn sign_semantic(&self, context: &'static [u8], message: &T) -> S;
```

Comment:
i think it would be somewhat better to call this `sign_digestible` or something, I think many readers won't know what is meant by `semantic`